src/status_im/data_store/realm/schemas/base/v5/account.cljs

```diff
@@ -0,0 +1,29 @@
+(ns status-im.data-store.realm.schemas.base.v5.account
+  (:require [taoensso.timbre :as log]))
+
+(def schema {:name       :account
+             :primaryKey :address
+             :properties {:address             :string
+                          :public-key          :string
+                          :updates-public-key  {:type     :string
+                                                :optional true}
+                          :updates-private-key {:type     :string
+                                                :optional true}
+                          :name                {:type :string :optional true}
+                          :phone               {:type :string :optional true}
+                          :email               {:type :string :optional true}
+                          :status              {:type :string :optional true}
+                          :debug?              {:type :bool :default false}
+                          :photo-path          :string
+                          :signing-phrase      {:type :string}
+                          :last-updated        {:type :int :default 0}
+                          :signed-up?          {:type    :bool
+                                                :default false}
+                          :network             :string
+                          :networks            {:type       :list
+                                                :objectType :network}
+                          :settings            {:type :string}}})
```

Comment:
why are you storing the string in account? but not creating new settings scheme?

Reply:
`settings` is a complex map (doesn't fit realm very well) so it's stored as serialized string.